.storybook/preview.tsx

```diff
@@ -14,7 +16,6 @@ ansiHTML.setColors({
   cyan: '5fb3b2',
 });
 
-chalk.enabled = true;
```

Comment:
Was this an intention deletion?

Reply:
I'm adding this back. It was a TypeScript error, but we added a ts-expect-error